docarray/document/mixins/mesh.py

```diff
@@ -40,3 +41,57 @@ def load_uri_to_point_cloud_tensor(
             self.tensor = np.array(mesh.sample(samples))
 
         return self
+
+    def load_uri_to_vertices_and_faces_chunk_tensors(self: 'T') -> 'T':
```

Comment:
```suggestion
    def load_uri_to_vertices_and_faces(self: 'T') -> 'T':
```

name is too long IMO (even though it stick to our old naming convention)